src/app/containers/ScheduleControls.tsx

```diff
@@ -13,14 +13,16 @@ import {
 } from '../modules/schedule/selectors';
 import {
     updateVisibleTime as setVisibleTime,
-    clearSelectedShifts
+    clearSelectedShifts,
+    selectSetOfShifts
```

Comment:
perhaps name `setSelectedShifts` (I've been using this convention in other places) ?